projects/plugins/jetpack/functions.opengraph.php

```diff
@@ -99,7 +99,7 @@ function jetpack_og_tags() {
 		$tags['og:title'] = wp_get_document_title();
 
 		$archive = get_queried_object();
-		if ( ! empty( $archive ) ) {
+		if ( ! empty( $archive ) && $archive instanceof WP_Term ) {
```

Comment:
`get_queried_object()` can return different objects depending on the context. Here, we expect to see WP_Term as we are working with archives, but apparently, we sometimes get WP_Post_Type instead. Addig a check for WP_Term seems like a solution.

Reply:
I think we can simplify this
```suggestion
		if ( $archive instanceof WP_Term ) {
```
`instanceof` will already do the right thing if it's null, false, or the like.

Reply:
Good point. Thanks